### PR TITLE
Attempt to add bebe_chr

### PR DESCRIPTION
**Pull request by MrPowers**

### CODE REVIEW

**Comment by MrPowers** [comment]
Anyone know how to make a DataFrame with a `CharType` column?

**Comment by alfonsorr** [comment]
There is no char type in spark, the function returns a string type
https://github.com/apache/spark/blob/02e74b298a98b579e95a22080832100fa4384712/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala#L2016

File: src/test/scala/org/apache/spark/sql/BebeFunctionsSpec.scala [hunk 1]
```diff
@@ -46,6 +49,27 @@ class BebeFunctionsSpec extends FunSpec with SparkSessionTestWrapper with Column
     }
   }
 
+  describe("bebe_chr") {
+    it("returns the ASCII character of a character") {
```
Comment (alfonsorr) [inline src/test/scala/org/apache/spark/sql/BebeFunctionsSpec.scala:53]: the behaviour is the opposite, you provide a number and returns a single character string with the letter. The expression to do it backwards is the Ascii

Comment (MrPowers) [reply src/test/scala/org/apache/spark/sql/BebeFunctionsSpec.scala:53]: Ah, thanks!

**Pull request closed without merging**